rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -428,6 +434,60 @@ def test_find_uniswap_v2_lp_token_price(inquirer, ethereum_manager, globaldb):
     assert price is not None
 
 
+@pytest.mark.vcr
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_find_sushiwap_v2_lp_token_price(database: 'DBHandler', inquirer_defi: 'Inquirer') -> None:
+    assert inquirer_defi.find_usd_price(asset=get_or_create_evm_token(
+        userdb=database,
+        evm_address=string_to_evm_address('0x397FF1542f962076d0BFE58eA045FfA2d347ACa0'),
+        chain_id=ChainID.ETHEREUM,
+        token_kind=TokenKind.ERC20,
+        protocol=CPT_SUSHISWAP_V2,
+    )).is_close('256957131.337105')
```

Comment:
I get the point of splitting them in multiple tests but since all of them check the same logic why not have it in one tests. You avoid setting the environment 4 times